packages/vitest/src/node/reporters/junit.ts

```diff
@@ -195,10 +207,31 @@ export class JUnitReporter implements Reporter {
 
   async writeTasks(tasks: Task[], filename: string): Promise<void> {
     for (const task of tasks) {
+      let classname = filename
+
+      const templateVars: ClassnameTemplateVariables = {
+        filename: task.file.name,
+        filepath: task.file.filepath,
+        suitename: this.options.suiteName ?? task.suite?.name ?? '',
```

Comment:
what should be a suitename if the test is declared at the top level? Currently, `suite` there is `undefined`. Is `''` (empty string) intended here?

Reply:
I'm not sure if empty string is the way to go here. I'm open to suggestions.

Reply:
`options.suiteName` is about `<testsuites name="...">`, so that shouldn't come first.

Also Jest's "suite name" concept is a bit different (and messed up) while ours simply use `<testsuite name=(filepath)>`, so we might not even include `suitename` here in `ClassnameTemplateVariables`.